Update json-editor

ikushum opened the pull request:
Updated JSON editor package and adjusted breaking changes.

thomas4019 commented:
@ikushum Can you explain more what the benefit of these changes are? I'd like to test myself as well so if you could provide some instructions to reproduce an issue or two that would be really helpful!

ikushum commented:
> @ikushum Can you explain more what the benefit of these changes are? I'd like to test myself as well so if you could provide some instructions to reproduce an issue or two that would be really helpful!

@thomas4019

**Short answer**: It has no benefit.

**Long Answer:** With this package, it was not possible to remove a value from a file type field (unless the field is an array) from the UI. So, to see if it has been addressed in the latest version, I updated the package and fixed all the breaking changes just to find out the issue still exists. However, since I updated and package, and fixed all the breaking changes, instead of stashing all the changes, I opened a PR.

thomas4019 reviewed (approved):
Nice work and thanks for the explanation! It's good to have us on the latest version
